auctions/tests/view_tests.py

```diff
@@ -64,6 +78,26 @@ def test_post_new_ask_with_0_offer(self, mock_messages):
         self.assertEqual(retreive_bid.offer, 0)
         self.assertEqual(retreive_bid.ask, 5)
 
+    @fudge.patch('auctions.views.messages')
+    def test_GET_POST_with_url_fragment(self, mock_messages):
+        mock_messages.is_a_stub()
+        url = 'https://github.com/codesy/codesy/issues/380#issue-163534117'
+        self.view.request = (fudge.Fake().has_attr(
+            POST={
+                'url': url,
+                'ask': 5,
+                'offer': 0,
+            })
+            .has_attr(user=self.user1))
+        self.view.post()
+
+        self.view.request = (fudge.Fake()
```

Comment:
Should also test if the non-fragmented url returns correct bid details.

e.g url= https://github.com/codesy/codesy/issues/380  should also return ask 5 offer 0.